docs/content/open-api/openapi.yaml

```diff
@@ -0,0 +1,1296 @@
+              type: number
+              example: 577000
+            timer_query_compile_stage_check_undefined_funcs_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 426000
+            timer_query_compile_stage_check_unsafe_builtins_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 14000
+            timer_query_compile_stage_resolve_refs_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 3000
+            timer_query_compile_stage_rewrite_comprehension_terms_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 329000
+            timer_query_compile_stage_rewrite_dynamic_terms_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 27000
+            timer_query_compile_stage_rewrite_expr_terms_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 40000
+            timer_query_compile_stage_rewrite_local_vars_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 187000
+            timer_query_compile_stage_rewrite_to_capture_value_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 1178000
+            timer_query_compile_stage_rewrite_with_values_ns:
+              description: '(Only returned if `instrument` is true.) *Description is forthcoming*'
+              type: number
+              example: 23000
+    200Provenance:
+      type: object
+      properties:
+        provenance:
+          type: object
+          properties:
+            version:
+              type: string
+              description: The version of this OPA instance
+            build_commit:
+              type: string
+              example: 4c6e524
+              description: The Git commit id of this OPA build.
+            build_timestamp:
+              type: string
+              description: 'When this OPA instance was built (in [ISO8601 format](https://www.w3.org/TR/NOTE-datetime))'
+            build_hostname:
+              type: string
+              description: The hostname where this instance was built.
+              example: 3bb58334a5a9
+            bundles:
+              type: object
+              description: A set of key-value pairs describing each bundle activated on the server.
+              properties:
+                authz:
```

Comment:
`authz` here is just an example name of a bundle, it could be named anything really.